R/build-news.R

```diff
@@ -299,6 +299,25 @@ tweak_news_heading <- function(x, versions, timeline, bs_version) {
       xml2::xml_add_sibling(date_nodes, .where = "after")
   }
 
+  ## one level down for BS4
+  if (bs_version > 3) {
+     x %>%
```

Comment:
As I found out rather stupidly, the order of these operations matters :see_no_evil: